src/components/SnapshotDetails/tabs/SnapshotPipelineRunsTab.tsx

```diff
@@ -56,8 +77,9 @@ const SnapshotPipelineRunTab: React.FC = () => {
     );
   }
 
-  if (LoadError) {
-    <StatusBox loadError={LoadError} loaded={loaded} />;
+  const errorMessage = getErrorMessage(LoadError);
```

Comment:
@aThorp96 I mentioned the wrong utility 🤦 we should use `getErrorState` here, something like this:

```tsx
  const errorMessage = getErrorState(LoadError, loaded, 'pipeline runs');
  if (errorMessage) {
    return errorMessage;
  }
```

could you please update - sorry about that 😕